### ISSUE TITLE
Errors when importing through Package Manager via git url

### ISSUE THREAD

**Issue by RicardoGuzmanVelasco** [BoundfoxStudios/fluentassertions-unity#3]
Hi!
First and foremost thanks on your very job having FluentAssertions on Unity. It's an essential to me and I have always managed to import it manually using the local cumbersome way.

I have tried to import it through Package Manager using the git url, both Unity 2019.4 and Unity 2020.1, some different versions. Having Test Framework up to date — currently, while last try, 1.1.22.
But after import it then two compile errors appears on console. I may miss something as I'm still not used to this way of remote import packages, which I'm loving btw.
These are both of them:

`Library\PackageCache\com.boundfoxstudios.fluentassertions@9468b214b4\Editor\FluentAssertions\Events\EventHandlerFactory.cs(27,36): error CS0246: The type or namespace name 'DynamicMethod' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.boundfoxstudios.fluentassertions@9468b214b4\Editor\FluentAssertions\Events\EventHandlerFactory.cs(36,13): error CS0246: The type or namespace name 'ILGenerator' could not be found (are you missing a using directive or an assembly reference?)`

A wider problem I think the package has is BoundfoxStudios.FluentAssertions being an Any Platforms asmdef. The way I see it is FluentAssertions should be an editor exclusive asmdef, shouldn't it?
Those two errors go away after I open the asmdef in the explorer and add "Editor" into include platforms array.
The same if set Api Compatibility Level to  .NET 4.x, which we musn't, I know. But it works! Just FYI.

So, long story short, what's the propoer way to install throuh Package Manager git url? And last but not least, is there anything I'm missing?
Thanks in advance.

**Comment by ManuelRauber** [comment]
Hi,

thanks for pointing that out. Strange, in my game, where I used it, it works without being an editor only package. However, when I installed it in a new project, I've got the same error as you've mentioned above.

I just updated the package so the Assembly is now an editor only assembly. 

The proper way of installing is the way you did it, it was just an error with the asmdef on my side. :) 

If you have any other issues, please let me know by creating a new issue. :-)

Edit: if you reinstalled the package, the fixed version number is `5.10.3-r4`
